setup.py

```diff
@@ -1,12 +1,16 @@
+"""
+bwctl-resources packaging config
+"""
 import os
 
 from setuptools import setup
 
 
 def read(filename):
+    """Read file's content"""
```

Comment:
Shall we rename function to `read_version()` and docstring to `Read version info from file`?

Reply:
Not relevant